code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -53,61 +55,148 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 /datum/clockwork_scripture/proc/run_scripture_effects() //The actual effects of the recital after its conclusion
 
+/datum/clockwork_scripture/channeled //Channeled scripture begins instantly but runs constantly
+	var/chant_invocation = "Ayy lmao"
+	var/chant_amount = 5 //Times the chant is spoken
+	var/chant_interval = 10 //Amount of deciseconds between times the chant is actually spoken aloud
+
+/datum/clockwork_scripture/channeled/recital()
+	if(!can_recite())
+		return 0
+	invoker << "<span class='brass'>You begin chanting a piece of scripture entitled \"[name]\".</span>"
+	slab.busy = "Chanting in progress"
+	return 1
+
+/datum/clockwork_scripture/channeled/run_scripture_effects()
+	if(!recital())
+		return 0
+	for(var/i = 0, i < chant_amount, i++)
```

Comment:
Would it be `for(var/i in 0 to chant_amount)` or `for(var/i in 1 to chant_amount)`?